Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Standard Configuration/configuration_with_lf_fem_and_octave.py

```diff
@@ -0,0 +1,759 @@
+            "smearing": 0,
+        },
+        "rr2": {
+            "RF_inputs": {"port": ("octave1", 1)},
+            "RF_outputs": {"port": ("octave1", 1)},
+            "intermediate_frequency": resonator_IF_q2,  # frequency at offset ch8
+            "operations": {
+                "cw": "const_pulse",
+                "readout": "readout_pulse_q2",
+            },
+            "time_of_flight": time_of_flight,
+            "smearing": 0,
+        },
+        "q1_xy": {
+            "RF_inputs": {"port": ("octave1", 2)},
+            "intermediate_frequency": qubit_IF_q1,  # frequency at offset ch7 (max freq)
+            "operations": {
+                "cw": "const_pulse",
+                "saturation": "saturation_pulse",
+                "x180": "x180_pulse_q1",
+                "x90": "x90_pulse_q1",
+                "-x90": "-x90_pulse_q1",
+                "y90": "y90_pulse_q1",
+                "y180": "y180_pulse_q1",
+                "-y90": "-y90_pulse_q1",
+            },
+        },
+        "q2_xy": {
+            "RF_inputs": {"port": ("octave1", 3)},
+            "intermediate_frequency": qubit_IF_q2,  # frequency at offset ch8 (max freq)
+            "operations": {
+                "cw": "const_pulse",
+                "saturation": "saturation_pulse",
+                "x180": "x180_pulse_q2",
+                "x90": "x90_pulse_q2",
+                "-x90": "-x90_pulse_q2",
+                "y90": "y90_pulse_q2",
+                "y180": "y180_pulse_q2",
+                "-y90": "-y90_pulse_q2",
+            },
+        },
+        "q1_z": {
+            "singleInput": {
+                "port": (con, fem, 7),
+            },
+            "operations": {
+                "const": "const_flux_pulse",
+            },
+        },
+        "q2_z": {
+            "singleInput": {
+                "port": (con, fem, 8),
+            },
+            "operations": {
+                "const": "const_flux_pulse",
+            },
+        },
+    },
+    "octaves": {
+        "octave1": {
+            "RF_outputs": {
+                1: {
+                    "LO_frequency": resonator_LO,
+                    "LO_source": "internal",
+                    "output_mode": "always_on",
+                    "gain": 0,
+                },
+                2: {
+                    "LO_frequency": qubit_LO_q1,
+                    "LO_source": "internal",
+                    "output_mode": "always_on",
+                    "gain": 0,
+                },
+                3: {
+                    "LO_frequency": qubit_LO_q2,
+                    "LO_source": "internal",
+                    "output_mode": "always_on",
+                    "gain": 0,
+                },
+            },
+            "RF_inputs": {
+                1: {
+                    "LO_frequency": resonator_LO,
+                    "LO_source": "internal",
+                },
+            },
+            "connectivity": (con, fem),
```

Comment:
I think that the elements don't match the default connectivity to the Octave:
RF3 -> (I=AO5 & Q=AO6)
Also since RF2 and RF3 share the same LO, I would keep the current wiring for Q1, but set Q2 XY drive to RF4 (AO 7 & 8) and Q2 flux on AO6 or AO9, WDYT?

Reply:
I'm not sure I follow the issue here. I think the confusion might be related to the mistake I made in the AO comments- in your comment you thought the correction would be to label AO5 as Q1 flux, but it should actually be Q2 XY I-quadrature. The correct labeling/wiring leads to:

AO1 & AO2 = Resonator
elements `rr1` and `rr2` refer to `{"port": ("octave1", 1)}`

AO3 & AO4 = Qubit 1 XY
element `q1_xy` refers to `{"port": ("octave1", 2)}`

AO5 & AO6 = Qubit 2 XY
element `q2_xy` refers to `{"port": ("octave1", 3)}`

This is the same configuration as the non-LF version and shouldn't require any non-default octave connectivity. Does this make sense?

Reply:
I don't remember the comment about the connectivity so let's say it is okay (it does look fine to be honest), however I don't like having qubit_LO_q1 and qubit_LO_q2 for the Octave ports RF2 and RF3 because they have to be equal since these ports are using the same mw synth. This is why I said that Q2 XY should be connected to RF4 instead and adjust the flux lines accordingly

Reply:
I understand now what you mean. I think it's a very valid point and should be changed, but since we should merge this one ASAP let's follow it up in a separate PR